source/includes/databases-collections/time-series.scala

```diff
@@ -0,0 +1,47 @@
+import org.bson.json.JsonWriterSettings
+import org.mongodb.scala._
+import org.mongodb.scala.bson.Document
+import org.mongodb.scala.model._
+
+import java.util.concurrent.TimeUnit
+import scala.concurrent.Await
+import scala.concurrent.duration.Duration
+
+object TimeSeries {
+
+  def main(args: Array[String]): Unit = {
+    val mongoClient = MongoClient("<connection string URI>")
+
```

Comment:
S: add code comments before each of these examples (you can probably use the same ones as [PHP](https://github.com/mongodb/docs-php-library/blob/master/source/includes/databases-collections/time-series.php))